[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-2 (#115637)
openshift/openshift-docs: modules/mirror-registry-localhost.adoc

[id="mirror-registry-localhost_{context}"]
= Mirroring on a local host with mirror registry for Red Hat OpenShift

[role="_abstract"]

> This procedure explains how to install the _mirror registry for Red{nbsp}Hat OpenShift_ on a local host by using the `mirror-registry` installer tool. By doing so, users can create a local host registry running on port 443 for the purpose of storing a mirror of {product-title} images.

Reviewer: ```suggestion
You can install the _mirror registry for Red{nbsp}Hat OpenShift_ on a local host by using the `mirror-registry` installer tool. By doing so, users can create a local host registry running on port 443 for the purpose of storing a mirror of {product-title} images.
```

.Procedure

> . Download the `mirror-registry.tar.gz` package for the latest version of the _mirror registry for Red{nbsp}Hat OpenShift_ found on the OpenShift console *Downloads* page.

Reviewer: Was this showing up as an error in the DITA tool? I think links are fine to keep in procedure modules, in which case just put this link back

$ podman login -u init \
  -p <password> \
  <host_example_com>:8443> \
  --tls-verify=false
----
+
where:
+

> `--tls-verify=false`:: You can avoid running `--tls-verify=false` by configuring your system to trust the generated rootCA certificates. See "Securing {quay}" and "Configuring the system to trust the certificate authority" for more information.

Reviewer: A few things:

1. When using description lists, always start the description with "Specifies". If that's hard to do without rewriting too much, just add a generic sentence in the beginning like "Specifies the `--tls-verify=false` flag." and then use the existing description after it.
2. In this case, it's easier to not use a description list at all:

```suggestion
+
You can avoid running `--tls-verify=false` by configuring your system to trust the generated rootCA certificates. See "Securing {quay}" and "Configuring the system to trust the certificate authority" for more information.
```
